modules/system/classes/MailManager.php

```diff
@@ -178,7 +178,12 @@ public function renderTemplate($template, $data = [])
 
         $css = MailBrandSetting::renderCss();
 
+        $disableInlineCss = false;
```

Comment:
Would it not be more appropiate to name this "$disableCssToInline"? Because we are not someohow preventing the layout to have inline styles, we are just toggling whether or not we want defined css classes converted to inline css.